.dockerignore

```diff
@@ -0,0 +1,4 @@
+.cache
```

Comment:
We don't need this file - please undo

Reply:
We need this file to prevent unnecessary file transfer between the Docker CLI and Docker Daemon when building the Docker Image. Here is documentation on how it works: https://docs.docker.com/engine/reference/builder/#dockerignore-file
This is the best practice when building Docker Images.